src/NUnitEngine/nunit.engine/Runners/MasterTestRunner.cs

```diff
@@ -105,7 +105,9 @@ protected bool IsPackageLoaded
         /// <returns>An XmlNode representing the loaded assembly.</returns>
         public XmlNode Load()
         {
-            LoadResult = _engineRunner.Load();
+            LoadResult = PrepareResult(_engineRunner.Load())
+                .Aggregate(TEST_RUN_ELEMENT, TestPackage.Name, TestPackage.FullName);
```

Comment:
Before the top-level xml-node was `test-suite`, now it is `test-run`. I guess that this could affect client code? (but looking at a resulting TestResult.xml from a execution it seems that this is already the case for the xml from a test execution, so I guess that this change will only affect the clients uses the engine directly, and not just rely on the console).

Reply:
The master test runner never returns the direct result of Load unless you call Load directly. Most times in the past it has only been called indirectly through Explore or Run so the LoadResult was immediately updated.

Looked at that way, this can only affect someone who is directly calling Load or Reload on MasterTestRunner, in which case they are already broken, or someone who has extracted our internal TestEngineRunner classes for their own use, since we don't expose the internal logic in the API.